docs/getting-started.md

```diff
@@ -10,15 +10,15 @@ cd my-condo
 
 # 2) Install dependencies and link workspaces
 yarn
-```
 
-# Create new application
+# 3) startup redis and postgres
+docker-compose up -d postgresdb redis
 
-```bash
-# 4) run create app command
-yarn createapp APPNAME
+# 4) create base .env file
+cp .env.example .env
 
-# 5) follow the README quick start (check ./apps/APPNAME/ folder)
+# 5) create databases for all apps and apps/.env files
+node ./bin/prepare.js
 ```
 
 [continue to some advanced topics](getting-started-adv.md)
```

Comment:
It this guide up to date with current implementation?

Reply:
It's in another PR. I want to sync it too